media-proxy/include/mesh/conn.h

```diff
@@ -92,6 +92,8 @@ class Connection {
     Result do_receive(context::Context& ctx, void *ptr, uint32_t sz,
                       uint32_t& sent);
 
+    Result transmit(context::Context& ctx, void *ptr, uint32_t sz);
```

Comment:
Don't make it public. This method has intentionally been made protected to allow calling it only from classes derived from Connection.

Add an additional public wrapper method to the emulated tx/rx to call `trasmit()`.